README.md

```diff
@@ -1237,3 +1237,32 @@ DELETE /api/v1/users/mary
 ```
 HTTP/1.1 200 OK
 ```
+
+# Upgrades
+
+Initiates an upgrade to the provided version. Once this endpoint is called [Horticulturalist](https://github.com/medic/horticulturalist) will take over and complete the installation.
+
+### Permissions
+
+`_admin` (CouchDB Administrator)
+
+## Example
+
+```
+POST /api/v1/upgrade
+{
+  "build": {
+    "namespace": "medic",
+    "application": "medic",
+    "version": "3.0.0-beta.1"
+  }
+}
+```
+
```

Comment:
@alxndrsn I know the code has already been written but afaict, I'm asking to remove features, not add new ones, because it's always more work and unlikely to be removed later.